Update Dockerfile to download voices

justinh-rahb opened the pull request:
This pull request updates the Dockerfile to include the necessary scripts for downloading voices. The `init.sh` script checks if the voices directory is empty and downloads the voices if necessary. This update ensures that the Docker container downloads the required files for voice models, making it easier for users to use the application.

matatonic commented:
Thanks Justin from the Hammer! (I'm in Toronto). I actually had the voices in the image originally and removed them... let me think about this before putting them back. Thanks for the arm changes!

justinh-rahb commented:
> Thanks Justin from the Hammer! (I'm in Toronto). I actually had the voices in the image originally and removed them... let me think about this before putting them back. Thanks for the arm changes!

Greetings @matatonic in The Big Smoke! Unfortunately the ARM-related changes probably need to go in a new PR, I've reverted that.

matatonic commented:
I've merged what I think was the intent of all these changes - thanks!
